spec/lib/payment_calculator/turnip_steps/ecf/uplift_payment_steps.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module UpliftPaymentsSteps
+  class DummyClass
```

Comment:
I think we should rename this to something more meaningful

```suggestion
  class ModuleTestHarness
```